schedule/RAID1_staging.yaml

```diff
@@ -0,0 +1,39 @@
+---
+name:           RAID1
+description:    >
+  Test for RAID1.
+  Installation of RAID1 using expert partitioner.
+vars:
+  RAIDLEVEL: 1
+conditional_schedule:
+  sle15_workarounds:
+    VERSION:
+      15-SP1:
+        - console/sle15_workarounds
+schedule:
+  - installation/bootloader
+  - installation/welcome
+  - installation/skip_registration
```

Comment:
On SLE15, 'accept_license' test module should be executed after 'welcome' (e.g. https://openqa.suse.de/tests/2813234#step/accept_license/1).

I guess here it should be done in the same way as in 'cryptlvm_minimal_x_staging.yaml' that was already made in the PR.

Reply:
nice catch. i dont know how i missed that. fixed